### PR TITLE
User service

### PR DESCRIPTION
**Pull request by Aphilosopher30**
# Description
Include a summary of changes and/or which issues have been fixed. Include context for features/logic/bugs that need particular attention.

# Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Closes # (issue)

# How Has This Been Tested?
- [ ] This has not been tested
- [ x] This has been tested

#Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -34,6 +34,10 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'figaro'
+gem 'omniauth-google-oauth2'
```
Comment (ztrokey) [inline Gemfile:38]: Awesome job implementing this!

File: app/controllers/welcome/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="jumbotron">
+  <div class="container">
+    <h1>Vote Local.</h1>
+    <p>
+        Do you want to be more involved in your community?
+        Have you ever felt like your vote didn't matter?
+        This is your first step to get involved in your local political scene.
+        Sign in to see who currently represents you!
```
Comment (ztrokey) [inline app/controllers/welcome/index.html.erb:8]: Is this just a placeholder for now?

**Review by ztrokey — COMMENTED** [review]
Can we add the coverage folder to the gitignore file? Let me know if you need any help. Unfortunately, I've had to do this myself far too many times haha.

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+class SessionsController < ApplicationController
```
Comment (EmmyMorris) [inline app/controllers/sessions_controller.rb:1]: Great job with setting up what is needed in this create method!

File: app/controllers/welcome/index.html.erb (hunk 2 above)
Comment (EmmyMorris) [reply app/controllers/welcome/index.html.erb:8]: We could definitely use the amazing 'About this Project' section @ztrokey wrote in the README here!

File: app/services/users_service.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class UsersService
```
Comment (EmmyMorris) [inline app/services/users_service.rb:1]: Great job with this user service file!

**Review by EmmyMorris — APPROVED** [review]
Great Job!

**Review by ztrokey — APPROVED** [review]
Looks awesome! Great work!

**Review by denverdevelopments — APPROVED** [review]

**Pull request merged**